src/netaddress.h

```diff
@@ -190,6 +192,7 @@ class CNetAddr
         }
 
         friend class CSubNet;
+        friend class CNetAddrHash;
```

Comment:
nit if you retouch, inverse the order of these two `friend class` declarements (both for alphabetical order and code order of the two classes, below)

Reply:
Done